core/domain/exp_services.py

```diff
@@ -421,6 +544,20 @@ def export_to_zip_file(exploration_id, version=None):
 
 
 def convert_state_dict_to_yaml(state_dict, width):
+    '''Converting state dict to yaml format
+
+    Args:
+        state_dict: A python dict.
```

Comment:
Explain both the type and what it is/represents. Ditto below. Your target audience here is new developers who don't have existing context.